rpi5.yaml

```diff
@@ -0,0 +1,260 @@
+desc: "Xen for Raspberry 5 comprehended"
+min_ver: "0.22"
+
+
+variables:
+  # Common build configuration
+
+  YOCTOS_WORK_DIR: "yocto"
+
+  DOM0_OS: "zephyr"
+
+  # Raspberry 5 build configuration
+
+  RPI_MACHINE: "raspberrypi5"
+  RPI_SOC_FAMILY: "bcm2712"
+
+  ZEPHYR_DOM0_MACHINE: "rpi_5_xen_dom0"
+  ZEPHYR_DOM0_TARGET: "zephyr/zephyr-dom0-xt"
+  ZEPHYR_DOM0_DIR: "zephyr"
+  ZEPHYR_DOM0_BUILD_DIR: "build"
+
+  DOMD_BUILD_DIR: "build/domd"
+  XT_XEN_DTB_NAME: "%{RPI_SOC_FAMILY}-%{RPI_MACHINE}-xen.dtb"
+
+  XT_KERNEL_BRANCH: "rpi-6.6.y"
+  XT_KERNEL_REV: "${AUTOREV}"
+
+  XT_DOMD_IMAGE: "image"
+
+  ZEPHYR_DOMD_MACHINE: "rpi_5_xen_domd"
+  ZEPHYR_DOMD_TARGET: "zephyr/samples/basic/blinky"
+  ZEPHYR_DOMD_DIR: "zephyr"
+  ZEPHYR_DOMD_BUILD_DIR: "build_d"
+
+  ZEPHYR_DOMU_MACHINE: "xenvm"
+  ZEPHYR_DOMU_TARGET: "zephyr/samples/synchronization"
+  ZEPHYR_DOMU_DIR: "zephyr"
+  ZEPHYR_DOMU_BUILD_DIR: "build_u"
+
+common_data:
+  # Sources used by domd
+  domd_sources: &DOMD_SOURCES
+    - type: git
+      url: "git://git.yoctoproject.org/poky"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.openembedded.org/meta-openembedded"
+      rev: "scarthgap"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-virtualization"
+      rev: "scarthgap"
+    - type: git
+      url: "git@github.com:dsemenets/meta-xt-common.git"
+      rev: "scarthgap"
+    - type: git
+      url: "git@github.com:dsemenets/meta-xt-rpi5.git"
+      rev: "init_new"
+    - type: git
+      url: "git@github.com:dsemenets/meta-xt-rpi5-prod-devel.git"
+      rev: "init_new"
+    - type: git
+      url: "git://git.yoctoproject.org/meta-selinux"
+      rev: "master"
+
+  # Zephyr sources
+  zephyr_sources: &ZEPHYR_SOURCES
+    - type: west
+      url: "https://github.com/GrygiriiS/zephyr.git"
+      rev: "zephyr-v3.6.0-xt-rpi5-dom0-wip"
+
+  # DomD configuration options for all yocto-based domains
+  domd_conf: &DOMD_CONF
+    - [SSTATE_DIR, "${TOPDIR}/../../../../../common_data/sstate"]
+    - [DL_DIR, "${TOPDIR}/../../../../../common_data/downloads"]
+
+    # Skip warning about missing "virtualization" distro feature
+    - [SKIP_META_VIRT_SANITY_CHECK, "1"]
+
+    # Use hypervisor console on all guests
+    - [SERIAL_CONSOLES, "115200;hvc0"]
```

Comment:
Is it true? I saw that we have some predefined `SERIAL_CONSOLES` in platforms, and I guess, we should override `SERIAL_CONSOLES` only if this is required.
So, the question is - is it required?

Reply:
Deleted